src/lmic/lorabase_as923.h

```diff
@@ -74,4 +74,12 @@ enum { DR_PAGE_AS923 = 0x10 * (LMIC_REGION_as923 - 1) };
 
 enum { AS923_LMIC_REGION_EIRP = 1 };         // region uses EIRP
 
-#endif /* _lorabase_as923_h_ */
\ No newline at end of file
+enum { AS923JP_LBT_5MS = 5 };          // microseconds of LBT time
+enum { AS923JP_LBT_DB_MAX = -80 };      // maximum channel strength
+
+#if !defined(AS923_TX_CAP)
+// AS923 v1.1, all channels face a 1% duty cycle.
+# define AS923_TX_CAP 100
+#endif
```

Comment:
Any reason why we shouldn't be good citizens and adopt 1% duty cycle on 1.0.2? It's easy to do, and I don't think it will cause any harm. But I'll leave as this for now.

Reply:
I don't want to be a bad guy basically :) However, it's helpful to increase duty cycle for a test purpose.  In particular, Japan regulation allows up to 10%.

Reply:
Understood. I adopted your change, more or less.  Once again, in order to facilitate future "single source builds" -- particularly for 1.0.2 + 1.1 single binary -- I tweaked slightly, but should be source compatible.